### PR TITLE
ticket to #57, tests added

### PR DESCRIPTION
**Pull request by mnjl1**
connect to #57 

tests added, but ...
Teacher class is still commented, 'cause Subject class  is not created (out of task scope).

### CODE REVIEW

File: src/test/java/io/github/externschool/planner/repository/profiles/PersonRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        entityManager.persist(person1);
+        entityManager.persist(person2);
+
+
+        List<Person> personList = this.personRepository.findAll();
+
+        assertThat(personList).isNotNull()
+                .hasSize(2).containsSubsequence(person1, person2);
+
+        assert(personList.get(0).equals(person1));
```
Comment (danilkuznetsov) [inline src/test/java/io/github/externschool/planner/repository/profiles/PersonRepositoryTest.java]: you already checked person 1 in previous assert

File: src/test/java/io/github/externschool/planner/repository/profiles/StudentRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+        List<Student> studentList = this.studentRepository.findAll();
+
+        System.out.println(studentList.get(0));
+
+        assertThat(studentList).isNotNull()
+                .hasSize(2)
+                .containsSubsequence(student1, student2);
+
+        assert(studentList.get(0).equals(student1));
```
Comment (danilkuznetsov) [inline src/test/java/io/github/externschool/planner/repository/profiles/StudentRepositoryTest.java]: you already checked person 1 in previous assert

File: src/test/java/io/github/externschool/planner/repository/profiles/StudentRepositoryTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        entityManager.persist(student1);
+
+        Student student2 = new Student();
+        student2.setFirstName("Vasia");
+        student2.setGender(Gender.MALE);
+        entityManager.persist(student2);
+
+        List<Student> studentList = this.studentRepository.findAll();
+
+        System.out.println(studentList.get(0));
```
Comment (danilkuznetsov) [inline src/test/java/io/github/externschool/planner/repository/profiles/StudentRepositoryTest.java]: it's redudant

**Comment by Benkoff** [comment]
@mnjl1 I think it's not a problem if you create an empty Subject class that formally is out of the scope. Otherwise you'll be unable to do anything with the class that actually is in.

File: src/test/java/io/github/externschool/planner/repository/profiles/StudentRepositoryTest.java (hunk 3 above)
Comment (Benkoff) [reply src/test/java/io/github/externschool/planner/repository/profiles/StudentRepositoryTest.java]: as i know printing in a test helps a lot while you're working on it but when it's done nobody reads what you've printed out there, your test passes or fails, that's all.

**Comment by mnjl1** [comment]
@Benkoff Empty class idea does not work: mapping problem, leaved it as is for now..

**Pull request merged**